huaweicloud/services/cdn/resource_huaweicloud_cdn_domain_rule.go

```diff
@@ -0,0 +1,948 @@
+package cdn
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API CDN GET /v1.0/cdn/configuration/domains/{domain_name}/rule
+// @API CDN PUT /v1.0/cdn/configuration/domains/{domain_name}/rules/full-update
+
+// Please apply for whitelist permission before using this resource.
+// The API used by this resource is an offline document and has no official website. So no documentation is provided yet.
+func ResourceCdnDomainRule() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCdnDomainRuleCreate,
+		ReadContext:   resourceCdnDomainRuleRead,
+		UpdateContext: resourceCdnDomainRuleUpdate,
+		DeleteContext: resourceCdnDomainRuleDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceCDNDomainRuleImportState,
```

Comment:
```suggestion
			StateContext: resourceCdnDomainRuleImportState,
```
keep the format union according to the above lines

Reply:
ok